docs/en/latest/how-to-build.md

```diff
@@ -21,351 +25,220 @@ title: Installing Apache APISIX
-
-  ```shell
-  sudo cpanm --notest Test::Nginx IPC::Run > build.log 2>&1 || (cat build.log && exit 1)
-  ```
-
-3. Clone the latest source code locally by using the forked out version.
-
-  ```shell
-  git clone https://github.com/openresty/test-nginx.git
-  ```
+```shell
+apisix start
+```
 
-4. There are two ways to run the tests.
+:::tip
 
-  1. Append the current directory to the perl module directory: `export PERL5LIB=.:$PERL5LIB` and then run `make test` command.
+Run `apisix help` to get a list of all available operations.
 
-  2. Specify the NGINX binary path by running `TEST_NGINX_BINARY=/usr/local/bin/openresty prove -Itest-nginx/lib -r t`.
+:::
 
-  <!--
-  #
-  #    In addition to the basic Markdown syntax, we use remark-admonitions
-  #    alongside MDX to add support for admonitions. Admonitions are wrapped
-  #    by a set of 3 colons.
-  #    Please refer to https://docusaurus.io/docs/next/markdown-features/admonitions
-  #    for more detail.
-  #
-  -->
+</TabItem>
+</Tabs>
 
-  :::note Note
-  Some of the tests rely on external services and system configuration modification. For a complete test environment build, you can refer to `ci/linux_openresty_common_runner.sh`.
-  :::
+## Installing etcd
 
-### Troubleshoot testing
+APISIX uses [etcd](https://github.com/etcd-io/etcd) to save and synchronize configuration.
 
-#### Configuring NGINX path
+etcd would be installed automatically if you have chosen the Docker or Helm install method above.
 
-The solution to the `Error unknown directive "lua_package_path" in /API_ASPIX/apisix/t/servroot/conf/nginx.conf` error is as shown below.
+If etcd is not installed you can install it as shown below.
 
-Ensure that OpenResty is set to the default NGINX, and export the path as follows:
+<Tabs
+  groupId="os"
+  defaultValue="linux"
+  values={[
+    {label: 'Linux', value: 'linux'},
+    {label: 'macOS', value: 'mac'},
+  ]}>
+<TabItem value="linux">
 
-* `export PATH=/usr/local/openresty/nginx/sbin:$PATH`
-  * Linux default installation path:
-    * `export PATH=/usr/local/openresty/nginx/sbin:$PATH`
-  * MacOS default installation path via homebrew:
-    * `export PATH=/usr/local/opt/openresty/nginx/sbin:$PATH`
+```shell
+ETCD_VERSION='3.4.18'
+wget https://github.com/etcd-io/etcd/releases/download/v${ETCD_VERSION}/etcd-v${ETCD_VERSION}-linux-amd64.tar.gz
+tar -xvf etcd-v${ETCD_VERSION}-linux-amd64.tar.gz && \
+  cd etcd-v${ETCD_VERSION}-linux-amd64 && \
+  sudo cp -a etcd etcdctl /usr/bin/
+nohup etcd >/tmp/etcd.log 2>&1 &
+```
 
-#### Running a single test case
+</TabItem>
 
-To run a specific test case, use the command below.
+<TabItem value="mac">
 
 ```shell
-prove -Itest-nginx/lib -r t/plugin/openid-connect.t
+brew install etcd
+brew services start etcd
 ```
 
-For more details on the test cases, see the [testing framework](https://github.com/apache/apisix/blob/master/docs/en/latest/internal/testing-framework.md) document.
+</TabItem>
+</Tabs>
```

Comment:
We shouldn't need to install etcd on MacOS as I didn't see installing APISIX for MacOS in the installation steps.

Reply:
We don't need to as it would be installed by the docker-compose file but users on mac can know this other way to install etcd. It is purely optional depending on how they installed.

Reply:
I checked the installation guide for the previous version. Since installing APISIX on MacOS using source code was added in the 2.13 version of the documentation, installing etcd on MacOS will be added. But the source installation of APISIX has been removed, so I think this step should be removed. Or take a look at other people's ideas.

Reply:
Yes, but mentioning etcd installation is always good as APISIX depends on it. Currently, installing using Docker will install etcd but even in that case, it might be good to mention.